Ginger/GingerCoreCommon/Repository/ApplicationModelLib/POMModelLib/ApplicationPOMModel.cs

```diff
@@ -41,6 +41,36 @@ public ApplicationPOMModel()
 
         private string mPageURL = string.Empty;
 
+        bool mIsPageLoadBusinessFlow;
+        public bool IsPageLoadBusinessFlow 
```

Comment:
as disscused- IsPageLoadBusinessFlow should be removed- radio btns setting should be done based on Enum Value and radio btns selectionChanged on page

Reply:
done